source/sdk/math.ooc

```diff
@@ -59,6 +59,13 @@ floor: extern(floorl) func ~Long (LDouble) -> LDouble
 	 maximumValue ::= static SHRT_MAX
 }
 
+extend Int64 {
+	modulo: func(divisor: This) -> This { 
+		result := this - (this / divisor) * divisor
+		result < 0 ? result + divisor : result
+	}
+}
+
```

Comment:
Okay, so exactly how does the behavior of this function differ from the one for `Int`?